src/as2fm/scxml_converter/scxml_entries/scxml_data_model.py

```diff
@@ -48,11 +48,13 @@ def from_xml_tree(xml_tree: ET.Element) -> "ScxmlDataModel":
             if is_comment(data_entry_xml):
                 prev_xml_comment = data_entry_xml.text.strip()
             else:
-                data_entries.append(ScxmlData.from_xml_tree(data_entry_xml, prev_xml_comment))
+                data_entries.append(ScxmlData.data_from_xml_tree(data_entry_xml, prev_xml_comment))
                 prev_xml_comment = None
         return ScxmlDataModel(data_entries)
 
-    def get_data_entries(self) -> Optional[List[ScxmlData]]:
+    def get_data_entries(self) -> List[ScxmlData]:
+        if self._data_entries is None:
+            return []
```

Comment:
I would rather enforce the `self._data_entries` to be a list (that is already done in `check_validity`, but could be enforced in the `__init__`, too!